prov/efa/src/efa_env.c

```diff
@@ -230,6 +237,8 @@ void efa_env_define()
 			"The maximum number of bytes that will be eagerly sent by inflight messages uses runting read message protocol (Default 307200).");
 	fi_param_define(&efa_prov, "use_sm2", FI_PARAM_BOOL,
 			"Use the experimental shared memory provider SM2 for intra node communication.");
+	fi_param_define(&efa_prov, "use_huge_page", FI_PARAM_BOOL,
+			"Whether EFA provider can use huge page memory for internal buffer. Using huge page memory has a small performance advantage, but can cause system to run out of huge page memory. By default: EFA provider will use huge page unless FI_EFA_FORK_SAFE is set to 1/on/true.");
```

Comment:
Max line length.

Reply:
Fixed

Reply:
Oh did you push this commit? It's still showing a single long line.

Reply:
Ah. I update the message in `efa_fork_support_initialize()`. Did not update here.

New revision updated here.